src/Authentication/LocalAuthenticator.php

```diff
@@ -186,4 +184,23 @@ public function validate(array $credentials, bool $returnUser=false)
             : true;
     }
 
+    /**
+     * Validates the user password
+     *
+     * @param User $user
+     * @param string $password
+     *
+     * @return bool
+     */
+    public function validate_password(User $user, string $password) : bool
```

Comment:
Typically all method names are camel-cased unless used for validation rules. 

```suggestion
    public function validatePassword(User $user, string $password) : bool
```